pack/fhv/fhvc.json

```diff
@@ -104,9 +104,359 @@
+    "clues": 1,
+    "code": "10555",
+    "double_sided": true,
+    "encounter_code": "the_silent_heath",
+    "encounter_position": 7,
+    "faction_code": "mythos",
+    "flavor": "The world is changing around us. I think, truly, for the better. - from the personal diary of Susan Pearl",
+    "illustrator": "Nele Diel",
+    "name": "Crystal Grove",
+    "pack_code": "fhvc",
+    "position": 55,
+    "quantity": 1,
+    "shroud": 2,
+    "text": "[action] Investigators at Crystal Grove spend 1 [per_investigator] clues, as a group: Add a Crystal Remains you control to the victory display.",
+    "traits": "Blight.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "The wreck of a once-great house stands precariously on the edge of a cliff. The decaying roofs look like an empty patchwork.",
+    "clues_fixed": true,
+    "clues": 1,
+    "code": "10556",
+    "double_sided": true,
+    "encounter_code": "the_silent_heath",
+    "encounter_position": 8,
+    "faction_code": "mythos",
+    "flavor": "The former orchard and farm has turned to powder from years of neglect. Nearly two decades have passed, yet nature has not reclaimed the opulent estate.",
+    "illustrator": "Pixoloid Studios",
+    "name": "Pearl Estate Ruins",
+    "pack_code": "fhvc",
+    "position": 56,
+    "quantity": 1,
+    "shroud": 3,
+    "text": "[action]: <b>Resign.</b> You leave the heath behind. Add all Crystal Remains you control to the victory display.",
+    "traits": "Ruins.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "The endless stretch of ash appears to be completely devoid of life.",
+    "clues": 1,
+    "code": "10557",
+    "double_sided": true,
+    "encounter_code": "the_silent_heath",
+    "encounter_position": 9,
+    "faction_code": "mythos",
+    "flavor": "The soft ash makes navigating the steep terrain especially treacherous.",
+    "illustrator": "Pixoloid Studios",
+    "name": "Ashen Slope",
+    "pack_code": "fhvc",
+    "position": 57,
+    "quantity": 1,
+    "shroud": 4,
+    "text": "<b>Forced</b> - When you reveal Ashen Slope: Test [agility] (4). If you fail, take 2 damage.",
+    "traits": "Blight.",
+    "type_code": "location"
+  },
+  {
+    "clues": 3,
+    "code": "10558",
+    "encounter_code": "the_silent_heath",
+    "encounter_position": 10,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
+    "name": "Salt Chamber",
+    "pack_code": "fhvc",
+    "position": 58,
+    "quantity": 1,
+    "shroud": 4,
+    "text": "<b>Revelation</b> - Put Salt Chamber into play adjacent to the location from which it was drawn\n<b>([codex] Ω)</b> <i>(page 21)</i>.\n[action] Investigators at Salt Chamber spend 1 [per_investigator] clues, as a group: Draw the top card of the stack beneath Salt Chamber. (Group limit once per round.)",
+    "traits": "Cave. Lair.",
+    "type_code": "location"
+  },
+  {
+    "encounter_code": "the_silent_heath",
```

Comment:
Please can you reorder these lines alphabetically? so 'clues', then 'code', etc? Thanks.

Reply:
sure, missed those